clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinator.java

```diff
@@ -123,6 +124,8 @@ public final class ConsumerCoordinator extends AbstractCoordinator {
 
     // hold onto request&future for committed offset requests to enable async calls.
     private PendingCommittedOffsetRequest pendingCommittedOffsetRequest = null;
+    // holds the offset metadata for assigned partitions to reduce remote calls thus speeding up fetching partition metadata
+    private final Map<TopicPartition, OffsetAndMetadata> partitionOffsetsCache;
```

Comment:
The term `offset` might be confused because we have the `offsets` fetching, and `offsets` committing. Please add `committed` in the variable name. Thanks.

Reply:
good suggestion, I'll do so.